src/policy/fees.cpp

```diff
@@ -392,6 +382,9 @@ void CBlockPolicyEstimator::processBlock(unsigned int nBlockHeight,
     for (unsigned int i = 0; i < entries.size(); i++)
         processBlockTx(nBlockHeight, entries[i]);
 
+    // Update best seen height after removals so they find the right mempool txs
```

Comment:
Can you clarify this comment?  I have been staring at this for a while and can't figure out what you meant.